source/features/minimize-upload-bar.css

```diff
@@ -0,0 +1,7 @@
+.rgh-has-upload-field .is-default .drag-and-drop {
+	display: none !important;
+}
+.rgh-has-upload-field .is-default textarea {
+	border-bottom-style: solid;
+	border-radius: 3px;
+}
```

Comment:
The correct CSS is the one in `clean-rich-text-editor`

https://github.com/sindresorhus/refined-github/commit/6f5b19862cd2a44ae1a64ff857dd45f4d2b90881

The code in `upload-button` was duplicate and wrong

Reply:
Ah damn, that's what [I had](https://github.com/sindresorhus/refined-github/pull/2157/commits/8e9a1d2bc311ba245f4335dc80069cd69a231857) before I simplified. Will revert.

Reply:
It’s best to keep the comment as well to justify the long selector.